paasta_tools/cli/utils.py

```diff
@@ -490,16 +490,21 @@ def run_paasta_serviceinit(subcommand, master, service, instancename, cluster, *
         delta = "--delta %s" % kwargs['delta']
     else:
         delta = ''
-    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s %s' % (
+    if 'stream' in kwargs and kwargs['stream']:
```

Comment:
Correction: stream is common to all commands, while verbose is "paasta status" only. I will leave the verbose flag unchanged while make stream a default argument if not explicitly specified.